invokeai/backend/generator/inpaint.py

```diff
@@ -377,3 +377,11 @@ def postprocess_size_and_mask(self, gen_result: Image.Image) -> Image.Image:
         )
 
         return corrected_result
+
+    def get_noise_like(self, like: torch.Tensor):
+        device = like.device
+        x = torch.randn_like(like, device=device)
+        if self.perlin > 0.0:
```

Comment:
We're not supporting perlin in `invokeai.app.invocations.noise`, at current. It should either be removed from here or support added back there.

Reply:
Good to know.

In this PR, this `get_noise_like` method is just shifted here from its superclass img2img to make it clear there is nothing remaining using the latter.

I suggest we leave this change in functionality for the next pass.